examples/khr-3hv/khr-3hv_continuous/controllers/supervisorsManager/khr3hvEnv.py

```diff
@@ -0,0 +1,206 @@
+        of the robot required are initialized here. 
+        For the observation we are using a time window of 5, to store the last 5 episode observations.
+        Observation:
+            Type: Box(10)
+            Num	Observation             Min(rad)      Max(rad)
+            0   Robot position x-axis   -inf          +inf      
+            1   Robot position y-axis   -inf          +inf
+            2   Robot position z-axis   -inf          +inf
+            3   Robot velocity x-axis    0            +7  
+            4	LeftAnkle               -inf          +inf
+            5	LeftCrus                -inf          +inf
+            6	LeftFemur               -inf          +inf
+            7	RightAnkle              -inf          +inf
+            8	RightCrus               -inf          +inf
+            9	RightFemur              -inf          +inf
+
+        Actions:
+            Type: Continuous
+            Num	BodyPost     Min      Max      Desc
+            0	LeftAnkle   -2.35    +2.35    Set the motor position from -2.35 to +2.35 
+            1	LeftCrus    -2.35    +2.35    Set the motor position from -2.35 to +2.35 
+            2	LeftFemur   -2.35    +2.35    Set the motor position from -2.35 to +2.35 
+            3	RightAnkle  -2.35    +2.35    Set the motor position from -2.35 to +2.35 
+            4	RightCrus   -2.35    +2.35    Set the motor position from -2.35 to +2.35 
+            5	RightFemur  -2.35    +2.35    Set the motor position from -2.35 to +2.35 
+
+        Reward: 
+            bonus for moving forward 
+            difference of current and pre vious positions
+
+        Starting State:
+            [0, 0, ..., 0]
+
+        Episode Termination:
+            Robot z axis smaller than 0.50 cm
+            Robot walked more that 15 m
+        """
+
+        super().__init__()
+        # Time window of 5
+        self.t = 5
+        # Number of observations
+        self.numObs = 10
+        # Total observations
+        self.obs = np.zeros(self.t * self.numObs)
+        # Lower and maximum values on observation space
+        lowObs = -np.inf * np.ones(self.t * self.numObs)
+        maxObs = np.inf * np.ones(self.t * self.numObs)
+        self.observation_space = Box(low=lowObs, high=maxObs, dtype=np.float64)
+        # Lower and maximum values on action space
+        lowAct = -2.356 * np.ones(6)
+        maxAct = 2.356 * np.ones(6)
+        self.action_space = Box(low=lowAct, high=maxAct, dtype=np.float64)
+
+        # Set up various robot components
+        self.motor_velocity = 2
+        self.robot = self.getSelf()
+        self.setup_agent()
+        self.motorPositionArr = np.zeros(6)
+        self.episodeScore = 0  # Score accumulated during an episode
+        self.prev_pos = self.robot.getPosition()[0]
+
+        # Logging parameters
+        self.distance_walk = 0
+        self.save_freq = wandbSaveFreq
+        self.counterLoging = 0
```

Comment:
Can we please change all variables in snake_case to CamelCase to be consistent throughout the example as well as the rest of the repository? Previous suggestion related to snake_case variable and this one is CamelCase.
```suggestion
        self.counterLogging = 0
```